swing/src/net/sf/openrocket/gui/dialogs/preferences/SimulationPreferencesPanel.java

```diff
@@ -116,6 +116,7 @@ public void actionPerformed(ActionEvent e) {
 				GeodeticComputationStrategy gcs = (GeodeticComputationStrategy) gcsCombo
 						.getSelectedItem();
 				gcsCombo.setToolTipText(gcs.getDescription());
+				gcsCombo.setSelectedItem(gcsCombo.getSelectedItem());
```

Comment:
Replace by `gcsCombo.repaint() // On some machines, the combobox did not visually update to the selected item`